src/Symfony/Component/Security/Http/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * `UserValueResolver` no longer implements `ArgumentValueResolverInterface`
+ * Calling the constructor of `DefaultLoginRateLimiter` with an empty secret is deprecated
```

Comment:
```suggestion
 * Deprecate calling the constructor of `DefaultLoginRateLimiter` with an empty secret
```

Reply:
Updated :+1: I'm not exactly sure how to word this [in 7.0](https://github.com/symfony/symfony/pull/51550/files#diff-6d49df895edd288cee25c95df434ae329795980f8024cdeb33ccd66ab65fb52cR306) :thinking: